.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ executors:
   windows_test: &windows_test_executor
     machine:
       image: "windows-server-2019-vs2019:stable"
-    resource_class: windows.xlarge
+    resource_class: windows.large
```

Comment:
I was checking if this change could reduce the size of our CI runners, since we use less RAM to build all the tests. That only worked for Windows though, the other runners still OOM'd when linking